### PR TITLE
Mali - DS-Unit-2-Regression-2

### PR DESCRIPTION
**Pull request by mohamad-ali-nasser**

### CODE REVIEW

**Comment by axrd** [comment]
# Module 01 - Log-Linear Regression, Feature Engineering

**Overall**: Nicely done! That's a darn good score. 

**Feedback**: Comment your code more! Go back and clean up your notebook where needed. It's a bit hard to follow along. Use text cells to mark the different parts, or elaborate on your code. 

## Load in the Data
- Glob is super useful, huh? 👍
- It would help to pre-emptively load them all into different dataframes. That way you can just merge them together as you go. But if you feel more comfortable using the pathnames, that also works. 

## Select Features 
- Proper merging of features 👍 
- Nice job evaluating feature importance 💯 

## Wrangling your Data
- Account for skew / log-transform 👍 
- Handle temporal data 👍 
- Great job making your code functional! 👍 👏 💯 

## Train/Val/Split
- Proper implementation, using train_test_split twice to create a validation set 👌 

## Fit a Model
- Yup. Nicely done. 

## Validation RMSLE (or RMSE)
- Yup. Now **boost** it 😁

**Comment by axrd** [comment]
# Module 02 - Gradient Boosting, More Feature Engineering

What happened? I don't see your assignment in your forked Repo? Gotta at least try! 

## More Feature Engineering
- ❓ 

## Implement Gradient Boosting
- ❓ 

[This is the best explanation of Gradient Boosting I've found so far](https://www.reddit.com/r/explainlikeimfive/comments/86dx3u/eli5_what_is_gradient_boosting_and_how_can_it_be/), in case you want to check it out.

Also, take a look at [Ryan's lecture notebook](https://colab.research.google.com/drive/1LAGRZ5jjO-35hHHTCOtO2VvxwRdwBy-o#scrollTo=zuzgmaBfYiiI) from today for a great reference of a possible solution. 

Lastly, were you able to figure out that for loop to iteratively merge different DFs together the way you wanted?
